packages/eslint-plugin-twenty-ts/dist/src/rules/no-state-useref.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
```

Comment:
we should not commit dist file. 
Let's actually add dist folder to gitignore

Reply:
`dist` in `eslint-plugin-twenty` is ignored, but `eslint-plugin-twenty-ts` was added and I figured, perhaps it serves a special purpose, hence the commits and updates